sdk/servicebus/service-bus/test/stress/pod.yaml

```diff
@@ -0,0 +1,39 @@
+kind: Pod
+apiVersion: v1
+metadata:
+  name: "%K8S_POD_NAME%"
+  namespace: richard
+spec:
+  restartPolicy: Never
+  volumes:
```

Comment:
I think you can delete this volume reference here and in the `volumeMounts` below. This pod doesn't need access to the service account token, and it seems the name is specific to this instance of the cluster anyway.

Reply:
You're correct, I meant to do that! 

And by seeing that you can probably guess how I derived this pod yaml in the first place...